src/test/java/com/google/api/codegen/testsrc/common/samples/sample_babble_about_book.yaml

```diff
@@ -0,0 +1,16 @@
+type: com.google.api.codegen.samplegen.SampleConfigProto
+config_schema_version: 1.0.0
```

Comment:
As per the spec, in all files:

s/`config_schema_version`/`schema_version`/g
s/`1.0.0`/`1.2.0`/g

Also, I would suggest naming all the files with `.sample.yaml`, eg `src/test/.../testsrc/common/samples/babble_about_book.sample.yaml`